### PR TITLE
 [54] Add Page Navigation to your Pages

### PR DESCRIPTION
**Pull request by kojusola**
## Task: `Uploaded signup page and included page navigation to all pages`
- ` created a signup page using only html and included page navigation to all pages, the screenshots of implementation can be found below:`
ContactUsNavigation0
Productpagenavigation0
productpagenavigation1
Signuppagenavigation0
signuppagenavigation1

### CODE REVIEW

**Comment by kojusola** [comment]
Removed the signup form page from the html forms folder.
Sorry for any inconveniences.

**Review by adaugochi — CHANGES_REQUESTED** [review]
You don't have signup.html file, Why are you linking it here? Please remove it

**Comment by kojusola** [comment]
Removed the sign-up page links.
Thanks.

File: submissions/adeolaFafemi/html-forms/contactUs.html [hunk 1]
```diff
@@ -13,8 +13,8 @@ <h3>Under 9k store</h3>
             <aside>
                 <ul>
                     <li><a href="#">Sign Up</a></li>
-                    <li><a href="#">Product Page</a></li>
-                    <li><a href="#">Contact Us</a></li>
+                    <li><a href="../HTML/index.html">Product Page</a></li>
+                    <li><a href="../html-forms/contactUs.html">Contact Us</a></li>
```
Comment (adaugochi) [inline submissions/adeolaFafemi/html-forms/contactUs.html:17]: I think just `contactUs.html` should work here

File: submissions/adeolaFafemi/HTML/index.html [hunk 2]
```diff
@@ -8,12 +8,13 @@
...
     <h3>Under 9k store</h3>
-    <section>
+       <section>
         <nav>
             <aside>
                 <ul>
-                    <li><a href="#">Home</a></li>
-                    <li><a href="#">Sign In</a></li>
+                    <li><a href="#">Sign Up</a></li>
+                    <li><a href="../HTML/index.html">Product Page</a></li>
```
Comment (adaugochi) [inline submissions/adeolaFafemi/HTML/index.html:16]: just `index.html` should work here

**Review by adaugochi — CHANGES_REQUESTED** [review]

**Comment by kojusola** [comment]
changed the navigation link.
thank you

**Review by adaugochi — APPROVED** [review]

**Pull request merged**